app/views/govuk_publishing_components/components/_admin_analytics.html.erb

```diff
@@ -6,5 +8,10 @@
 
   ga('create', 'UA-26179049-6', '<%= ENV['GOVUK_APP_DOMAIN'] %>');
   ga('set', 'anonymizeIp', true);
+
+  <% if user_organisation %>
+    ga('set', 'dimension8', "<%= user_organisation.presence || '(not set)' %>");
```

Comment:
This is great as it's needed for https://trello.com/c/32e3xgj4. But we need to make `dimension8` customisable, as every GA dashboard will return a different dimension depending on how many are added. In the case of content-publsiher may be the only one, hence `dimension1`